green/output.py

```diff
@@ -174,14 +181,14 @@ def write(self, text):
                 self.coverage_percent = int(percent_str)
         self.stream.write(text)
 
-    def writelines(self, lines):
+    def writelines(self, lines: Iterable[str]) -> None:
```

Comment:
FYI `Iterable[str]` is double edged as typing is concerned because strings are iterable lists of strings, so this typing will not catch if a caller is passing a plain string. This is an open 'bug' in mypy, but still better than having not typing at all.